package.json

```diff
@@ -91,6 +92,7 @@
     "babel": "5.8.38",
     "babel-core": "5.8.38",
     "babel-loader": "5.3.2",
+    "babel-polyfill": "6.9.1",
```

Comment:
Should we be using some v5 of this plugin? Is it compatible with our current Babel 5 setup?